lib/adal-aad-node/src/models/account.ts

```diff
@@ -34,6 +34,11 @@ export enum AzureAuthType {
     DeviceCode = 1
 }
 
+export enum AccountType {
+    Microsoft = 0,
```

Comment:
Consider using the string values here instead so you don't have to hardcode those separately?